flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -63,6 +73,14 @@ class FlxGamepad implements IFlxDestroyable
 	 * Helper class to get the justMoved, justReleased, and float values of analog input.
 	 */
 	public var analog(default, null):FlxGamepadAnalogList;
+	/**
+	 * Helper class to get the float values of motion-sensing input, if it is available
+	 */
+	public var motion(default, null):FlxGamepadMotionValueList;
+	/**
+	 * Helper class to get the float values of mouse-like pointer nput, if it is available
+	 */
+	public var pointer(default, null):FlxGamepadPointerValueList;
```

Comment:
Yeah probably! 

As a quick aside -- is it possible to override Flixel's mouse input with another source, so you could use a gamepad analog stick, or wiimote pointer input, or even keyboard presses, etc, to drive the mouse, rather than native mouse input? Would be super swell for accessibility.